### ISSUE TITLE
Resolution!

### ISSUE THREAD

**Issue by brayStorm** [jshridha/docker-blueiris#16]
How can we adjust the resolution. 
Can winecfg be run on initial install?

**Comment by MapGuy11** [comment]
I recommend doing /vnc.html when you open the noVNC console

**Comment by jcollie** [comment]
Building a new container based on the original seems to have done the trick for me. I'm just doing some testing - I don't even have any cameras yet - but this seems to be working.

```
FROM jshridha/blueiris
RUN sed -i s/800x600/1280x900/ /etc/supervisor/conf.d/supervisord.conf
```

**Comment by SHerms** [comment]
jcollie is correct. I modified that file and built another image. Figured this out awhile ago, just never got around to posting. Basic testing with 2 4k cameras seemed to work fine.

Here are instructions for someone who doesn't know how to build a new image with these changes.

First download the repo.
`git clone https://github.com/jshridha/docker-blueiris.git`
Move in to the folder and make changes 
`cd docker-blueiris/`
`sed -i s/800x600/1280x900/ /etc/supervisor/conf.d/supervisord.conf`
Build new container
`docker build -t blueiris2 .`
Once that is done you can start a new container with that image.
```
docker run -d \
  --name="BlueIris2" \
  -p 8080:8080 \
  -p 5900:5900 \
  -p 81:81 \
  -v ~/docker/blueiris2:/root/prefix32:rw \
  blueiris2
```

**Comment by SHerms** [comment]
I just created a pull request that changes the resolution to 1024x768
https://github.com/jshridha/docker-blueiris/pull/18

**Comment by Kosh42** [comment]
I'm almost there, but
`sed -i s/800x600/1280x900/ /etc/supervisor/conf.d/supervisord.conf`
give me:

`sed: can't read /etc/supervisor/conf.d/supervisord.conf: No such file or directory`

Any pointers for a newbie (Linux after a long break and Docker starting this morning). This is both running it as a user and sudo.

UPDATE: Think I sorted it. Edited the supervisord.conf file in the Git clone directory in a text editor to be the correct resolution. Worked a charm.

**Comment by jshridha** [comment]
This will be fixed once #27 is merged shortly.

**Comment by Dinth** [comment]
Has this been merged? 
I dont think it works for me, i also tried to change the resolution manually using the env variable RESOLUTION set to 1920x1080x24 and even that doesnt work
